pyvista/plotting/renderer.py

```diff
@@ -923,19 +965,25 @@ def add_axes_at_origin(
         >>> _ = pl.add_mesh(pv.Sphere(center=(2, 0, 0)), color='r')
         >>> _ = pl.add_mesh(pv.Sphere(center=(0, 2, 0)), color='g')
         >>> _ = pl.add_mesh(pv.Sphere(center=(0, 0, 2)), color='b')
-        >>> _ = pl.add_axes_at_origin()
+        >>> _ = pl.add_axes_marker()
         >>> pl.show()
 
         """
-        self._marker_actor = create_axes_marker(
-            line_width=line_width,
+        self._marker_actor = AxesActor(
+            x_label=x_label,
+            y_label=y_label,
+            z_label=z_label,
+            labels_off=labels_off,
             x_color=x_color,
             y_color=y_color,
             z_color=z_color,
-            xlabel=xlabel,
-            ylabel=ylabel,
-            zlabel=zlabel,
```

Comment:
Please make sure that you properly deprecate the usage of these keyword arguments. You'll need to capture additional `kwargs`, check for the presence of the old name and issue a deprecation warning if present. 

We have implemented a helper for this too:

https://github.com/pyvista/pyvista/blob/76aa761dac8dddf9995a49872399e4f9a406ecf8/pyvista/core/utilities/misc.py#L12

There are a few examples to model after. One example being:

https://github.com/pyvista/pyvista/blob/76aa761dac8dddf9995a49872399e4f9a406ecf8/pyvista/core/grid.py#L546-L552